tests/testing-resources/include/aws/testing/MemoryTesting.h

```diff
@@ -108,6 +110,19 @@ class AWS_TESTING_API ExactTestMemorySystem : public BaseTestMemorySystem
 
 };
 
+class AWS_TESTING_API CRTMemTracerMemorySystem : public Aws::Utils::Memory::MemorySystemInterface
+{
+public:
+    CRTMemTracerMemorySystem();
+    void Begin() override {}
+    void End() override {}
```

Comment:
I guess no allocations are allowed before Begin() and after End()

Reply:
sorry not following whats needs to be done here? these are optional calls for static setup and we dont do any static setup so we can skip them